src/main/java/tetris/KeyBindings.java

```diff
@@ -23,6 +23,11 @@ public KeyBindings() {
         bindings.put(KeyCode.SPACE, "HARD_DROP");
     }
 
+    /**
+     * Return names of all bindings
+     *
+     * @return
```

Comment:
I think it will generate checkstyle warns when you dont specify the return variable.